src/models/propulsion/FGBrushLessDCMotor.cpp

```diff
@@ -0,0 +1,242 @@
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+ Module:       FGBrushLessDCMotor.cpp
+ Autor         Paolo Becchi
+ 1st release   1/1/2022
+ Purpose:      This module models an BLDC electric motor
+
+  ------------- Copyright (C) 1999  Paolo Becchi (pbecchi@aerobusinees.it) -------------
+
+  This program is free software; you can redistribute it and/or modify it under
+  the terms of the GNU Lesser General Public License as published by the Free
+  Software Foundation; either version 2 of the License, or (at your option) any
+  later version.
+
+  This program is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+  FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+  details.
+
+  You should have received a copy of the GNU Lesser General Public License along
+  with this program; if not, write to the Free Software Foundation, Inc., 59
+  Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
+  Further information about the GNU Lesser General Public License can also be
+  found on the world wide web at http://www.gnu.org
+
+FUNCTIONAL DESCRIPTION
+--------------------------------------------------------------------------------
+Following code represent a new BrushLess DC motor to be used as alternative 
+to basic electric motor. 
+BLDC motor code is based on basic "3 constant motor equations"
```

Comment:
Do you have an online link/reference for the "3 constant motor equations"?

Reply:
not realy.....